src/beautifiers/js-beautify.coffee

```diff
@@ -9,7 +9,6 @@ module.exports = class JSBeautify extends Beautifier
     XML: true
     Handlebars: true
     Mustache: true
-    Marko: true
```

Comment:
Hey @Glavin001, sorry for not responding earlier, but I took a vacation. 

> Does js-beautify handle Marko language well enough?

I would say that, no, `js-beautify` does not handle the Marko v3 syntax well enough. Marko allows HTML attributes to be arbitrary JavaScript expressions and that will often completely throw off `js-beautify` and likely break the code.